backend/workflow_manager/endpoint/exceptions.py

```diff
@@ -69,3 +69,11 @@ class BigQueryTableNotFound(APIException):
         "Please enter correct correct bigquery table in the form "
         "{table}.{schema}.{database}."
     )
+
+
+class UnstractDBException(APIException):
```

Comment:
One doubt here, How this exeption came here? Will it be along DBconnector class ?

Reply:
No @muhammad-ali-e , as discussed with @chandrasekharan-zipstack , this will be handled here itself.
Since connector is a library, it will have exception like this https://github.com/Zipstack/unstract/pull/330/files/12028aaf13b7774bac79053c40d83783530f1294#diff-727c4e814179228a7201310d1c3c57ed32faa51f68b719a3a7220f64069bc54a